### ISSUE TITLE
action runs under a different user

### ISSUE THREAD

**Issue by ihulsbus** [haya14busa/action-bumpr#42]
Hi, I am using the action in my workflow in a dry-run mode to just determine the next version that needs to be built.
```
- name: bump version
  id: bump
  uses: haya14busa/action-bumpr@v1
  with:
    default_bump_level: patch
    dry_run: true
```
Unfortunately as the action is docker based and the [Dockerfile ](https://github.com/haya14busa/action-bumpr/blob/master/Dockerfile) does not contain a `USER` directive, the action runs under UID `1000` (root). 
This, for some reason seems to have as effect that all ownership of the .git folder change to UID `1000`, causing my next action with git to fail: 
```
- name: bump chart version
  run: |
    export chartVersion=$(cat Chart.yaml | grep '^version:' | awk '{ print $2 }')
    sed -i '/version:/ s/'${chartVersion}'/'${{ steps.bump.outputs.next_version }}'/' Chart.yaml
      
- name: commit bump
  run: |
    git add Chart.yaml
    git commit -m "bump chart version"
    git push
```
The `commit bump` task results in a `error: insufficient permission for adding an object to repository database .git/objects`

Would it be possible to change the action so it runs as a non-root user? Preferably UID `1001` as that is the default for the github runners?

```
user: runner
id: uid=1001(runner) gid=127(docker) groups=127(docker),4(adm),101(systemd-journal)
```

**Comment by haya14busa** [comment]
PR is welcome.

**Comment by ihulsbus** [comment]
Resolved in PR #43. It has been tested and permissions are now consistent when compared to before and  after using the bumpr action.
